src/pages/workspaces/workspace/analysis/_testData/testData.js

```diff
@@ -0,0 +1,243 @@
+import _ from 'lodash/fp'
+import * as Utils from 'src/libs/utils'
+import { tools } from 'src/pages/workspaces/workspace/analysis/notebook-utils'
+import {
+  defaultGceBootDiskSize, defaultGceMachineType, defaultGcePersistentDiskSize, defaultLocation, defaultPersistentDiskType, runtimeStatuses
+} from 'src/pages/workspaces/workspace/analysis/runtime-utils'
+import { v4 as uuid } from 'uuid'
+
+
+const defaultGoogleWorkspaceNamespace = 'test-ws'
+
+//this is important, so the test impl can diverge
+export const testDefaultLocation = defaultLocation
```

Comment:
How does it work?

Reply:
Not sure what you mean? I'm not tied to doing this, I just thought that it would be good to not use the default from the actual `src` code, and instead define a variable in the `test` code.

Reply:
I found the comment really confusing. Honestly, I'm still not sure what the point is.